### PR TITLE
Documentation for k8s-docker-provisioner including kube-up functionality

### PR DESCRIPTION
**Pull request by zreigz**

### CODE REVIEW

**Comment by zreigz** [comment]
@cheld @batikanu @f-higashi @taimir @kenan435 
Please review

**Comment by batikanu** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,87 +1,107 @@
...
-
-Provision DevStack.
+### Install OpenStack CLI tools
 
+- heat >= 0.9.0
+- nova >= 3.2.0
 ```
-cd ~/your_workspace/devstack-vagrant/
-DEVSTACK_MEM=10024 DEVSTACK_CPUS=6 vagrant up --provider libvirt
+ sudo pip install -U --force python-heatclient
```
Comment (cheld) [inline README.md]: is the force option really needed? I would rather remove it

File: README.md [hunk 2]
```diff
@@ -1,87 +1,107 @@
...
+
+ sudo pip install -U --force python-novaclient
 ```
 
-Download custom image from Google drive: <https://drive.google.com/file/d/0B6KOuPCy8tK1aDRCaE16N1hkcUU/view>
```
Comment (cheld) [inline README.md]: Kenan has put the image to the release tab of gitub. Can you adjust the link?

File: README.md [hunk 3]
```diff
@@ -1,87 +1,107 @@
...
 ```
 
-## Verification
+### Prepare Openstack image
 
-From your client computer:
+You need Centos image for Openstack with installed Docker and docker-bootstrap service. More info you can find in separate
+project which is dedicated in this purpose: [k8s-nodeos-builder](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/k8s-nodeos-builder)
+
+You can also download prepared Centos image from Google drive: <https://drive.google.com/file/d/0B6KOuPCy8tK1aDRCaE16N1hkcUU/view>
```
Comment (cheld) [inline README.md]: Kenan has put the image to the release tab of gitub. Can you adjust the link?

File: README.md [hunk 4]
```diff
@@ -1,87 +1,107 @@
...
-     --template-file kubecluster.yaml \
-     kubernetes_stack
+sudo cp kubectl /usr/local/bin
 ```
 
-## Verification
+### Prepare Openstack image
 
-From your client computer:
+You need Centos image for Openstack with installed Docker and docker-bootstrap service. More info you can find in separate
```
Comment (cheld) [inline README.md]: The provisioning works on any operating system that has a Docker and Docker-bootstrap service installed. 

You can download such a prepared image from ...

If you want to build your own image you can use this project .... bla bla

More info....

File: README.md [hunk 5]
```diff
@@ -1,87 +1,107 @@
-# heat-docker-kubernetes
+# Getting started with k8s-docker-provisioner
 
-Steps:
+This guide will take you through the steps of deploying Kubernetes to Openstack using docker. The primary concept is described here: [docker-multinode](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/kubernetes/blob/master/docs/getting-started-guides/docker-multinode/master.md)
 
-1. Deploy a Kubernetes cluster on DevStack using Heat
+The default OS is CentOS 7.
```
Comment (cheld) [inline README.md]: Please remove CentOS from this doc. Instead we should explain the Docker bootstrap concept. But I would suggest to do this task later

**Comment by cheld** [comment]
PTAL

File: README.md (hunk 4 above)
Comment (cheld) [inline README.md]: I assume the image name / ID must be specified somewhere.

**Comment by zreigz** [comment]
PTAL

File: README.md [hunk 6]
```diff
@@ -1,87 +1,125 @@
...
-Create and register a keypair.
+### Prepare Openstack image
+
+The provisioning works on any operating system that has a Docker >= 1.10 and Docker-bootstrap service installed. This service is used to
+run flannel network inside of Docker containers themselves. Service should execute following command:
 
 ```
-ssh-keygen -f ~/.ssh/id_rsa -P ''
-nova keypair-add keypair1 --pub-key ~/.ssh/id_rsa.pub
+docker daemon \
```
Comment (cheld) [inline README.md]: This is confusing. The reader will try to execute the docker command. I suggest to remove and leave a TODO.

Later, I would suggest a separate document that describes the docker bootstrap concept in detail and the manual steps. Maybe this document should be part of the other project.

**Comment by cheld** [comment]
One comment. PTAL

File: README.md (hunk 6 above)
Comment (zreigz) [inline README.md]: done

**Comment by zreigz** [comment]
PTAL

**Comment by cheld** [comment]
LGTM

You can merge together with the code

**Pull request merged**